tests/base/dataset_test.py

```diff
@@ -185,7 +185,7 @@ def test_dataset_infer_cat_features_max_categorical_ratio(diabetes_df):
 
 def test_dataset_label(iris):
     args = {'df': iris,
-            'label': 'target'}
+            'label_name': 'target'}
```

Comment:
What about test that tests the new label : pd.Series functionality?

Reply:
the `from_numpy` using it, so it tested implicitly

Reply:
fine, also added tests